### PR TITLE
Update log-files.md

### PR DESCRIPTION
**Pull request by pepieti**
Added note to check the timestamp of media logs. Also added that media logs are required for call related issues. Experience from IcMs has shown many times customers send media logs that are months before they reproduced the issue.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/log-files.md [hunk 1]
```diff
@@ -85,11 +85,11 @@ The debug logs are automatically downloaded to the following folders.
 Media logs
 ---------------------------
 
-Media logs contain diagnostic data about audio, video, and screen sharing in Teams meetings. They are required for support cases only upon request and can only be inspected by Microsoft. 
+Media logs contain diagnostic data about audio, video, and screen sharing in Teams meetings. They are required for support cases that are related to call related issues.
```
Comment (JohanFreelancer9) [inline Teams/log-files.md:88]:
```suggestion
Media logs contain diagnostic data about audio, video, and screen sharing in Teams meetings. They are required for support cases that are linked to call-related issues.
```

Comment (JohanFreelancer9) [reply Teams/log-files.md:88]: Suggestion to avoid the repetitive use of "related".

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@SerdarSoysal PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**